calico_containers/pycalico/datastore.py

```diff
@@ -341,8 +341,11 @@ def add_ip_pool(self, version, pool):
         # If IP in IP is enabled on the pool, ensure that it is enabled
         # globally.
         if pool.ipip:
-            result = self.etcd_client.read(IP_IN_IP_PATH)
-            if result.value != IP_IN_IP_ENABLED:
+            try:
```

Comment:
Could you add a comment describing the purpose of this try/except, and why we perform a read first at all?